src/integrations/watchers/indexable-post-watcher.php

```diff
@@ -162,7 +162,7 @@ protected function is_post_indexable( $post_id ) {
 	 */
 	public function build_indexable( $post_id ) {
 		// Bail if this is a multisite installation and the site has been switched.
-		if ( \is_multisite() && \ms_is_switched() ) {
+		if ( $this->is_multisite_and_switched() ) {
```

Comment:
Perhaps it is better to add this to the `site-helper` class and add the site-helper as a dependency to this class.

Reply:
I've considered that, but in this case it's just needed one single time. If we need this at more locations we should move it to an helper. But for now I want to be pragmatic.